src/components/library/LibraryControlBar.tsx

```diff
@@ -194,6 +227,51 @@ const LibraryControlBar: React.FC<Props> = (props: Props) => {
             {getFilterProgressText()} <DownOutlined />
           </Button>
         </Dropdown>
+        <Dropdown
+          className={styles.libraryTypeDropdown}
+          overlay={
+            <Menu
+              onClick={(e: any) =>
+                props.setLibraryTypes(e.item.props['data-value'])
+              }
+            >
+              <Menu.Item key={LibraryType.Grid} data-value={LibraryType.Grid}>
+                Grid
+              </Menu.Item>
+              <Menu.Item key={LibraryType.List} data-value={LibraryType.List}>
+                List
+              </Menu.Item>
+            </Menu>
+          }
+        >
+          <Button>
+            {getLibraryTypeText()} <DownOutlined />
+          </Button>
+        </Dropdown>
+        <Dropdown
+          className={styles.libraryTypeDropdown}
+          overlay={
+            <Menu
+              onClick={(e: any) =>
+                props.setLibrarySort(e.item.props['data-value'])
+              }
+            >
+              <Menu.Item key={LibrarySort.Asc} data-value={LibrarySort.Asc}>
+                Ascending
+              </Menu.Item>
+              <Menu.Item key={LibrarySort.Desc} data-value={LibrarySort.Desc}>
+                Descending
+              </Menu.Item>
+              <Menu.Item key={LibrarySort.Title} data-value={LibrarySort.Title}>
+                Title
+              </Menu.Item>
```

Comment:
Maybe I missed a thread elsewhere, but why are we adding a sort by number unread? I don't see the use-case for that.

I do think this can be used to change between sorting by title A-Z and Z-A.

Reply:
I personally use the number unread so to catchup series with most number of chapters available. 

Library sort with Title desc can be implemented easily.
Let me know if you wanna keep `number unread sort` or not, I'll make other changes in meantime.

Reply:
Having the num unread sort is fine with me, should just update the display text for the options.

Reply:
Updated Text correctly